src/sagemaker_inference/default_handler_service.py

```diff
@@ -63,4 +63,4 @@ def initialize(self, context):
         else:
             os.environ[PYTHON_PATH_ENV] = code_dir_path
 
-        self._service.validate_and_initialize(model_dir=model_dir)
+        self._service.validate_and_initialize(model_dir=model_dir, context=context)
```

Comment:
General comments:
1. Please update the `README.md` file for this change
2. Please squash your commits. 
3. Also refer to the commit message style here: https://github.com/aws/sagemaker-inference-toolkit/blob/master/CONTRIBUTING.md#committing-your-change